docs/docs/reference/release-notes/migrating-from-v2-to-v3.md

```diff
@@ -0,0 +1,570 @@
+---
+title: Migrating from v2 to v3
+---
+
+Looking for the v2 docs? [Find them here.](https://v2.gatsbyjs.com)
+
+> This document is a work in progress. Have you run into something that's not covered here? [Add you changes to GitHub!](#link-to-github)
+
+## Introduction
+
+This is a reference for upgrading your site from Gatsby v2 to Gatsby v3. TODO
+
+> If you want to start fresh, run `npm init gatsby` or `yarn create gatsby` in your terminal.
+
+## Why you should upgrade
+
+This documentation page covers the _how_ of migrating from v2 to v3. If you're curious what's new, read the [v3.0 release notes](#link-to-release-notes).
+
+## What we'll cover
+
+short list of contents, point to sidebar at the right
+
+## Updating Your Dependencies
+
+First, you need to update your dependencies.
+
+### Update Gatsby version
+
+You need to update your `package.json` to use the latest version of Gatsby.
+
+```json:title=package.json
+{
+  "dependencies": {
+    "gatsby": "^3.0.0"
+  }
+}
+```
+
+Or run
+
+```shell
+npm install gatsby@latest
+```
+
+### Update Gatsby related packages
+
+Update your `package.json` to use the latest version of Gatsby related packages. You should upgrade any package name that starts with `gatsby-*`. Note, this only applies to plugins managed in the [gatsbyjs/gatsby](https://github.com/gatsbyjs/gatsby) repository. If you're using community plugins, they might not be upgraded yet. Many plugins won't need updating so they well might keep working (if not, please check their repository for the current status). You can run an npm script to see all outdated dependencies.
+
+```shell
+npm outdated
+```
+
+Compare the "Wanted" and "Latest" versions and update their versions accordingly. For example, if you have this oudated version:
+
+```shell
+> npm outdated
+
+Package                  Current   Wanted  Latest  Location
+gatsby-plugin-sharp      2.14.1    2.14.1  3.0.0   test
+```
+
+Install the new package with:
+
+```shell
+npm install gatsby-plugin-sharp@latest
+```
+
```

Comment:
Despite the fact we default to npm is docs, I think it's worth including the yarn command here, as the workflow is completely different. The command is:
```shell
yarn upgrade-interactive --latest
```
...and then they need to select the gatsby packages to upgrade.